Git/gitconfig.sublime-syntax

```diff
@@ -0,0 +1,210 @@
+%YAML 1.2
+---
+# Syntax based on documentation here:
+# https://git-scm.com/docs/git-config#_syntax
+
+name: Git Config
+file_extensions:
+  - gitconfig                # /etc/gitconfig
+  - .gitconfig               # ~/.gitconfig
+first_line_match: ^\[core\]  # .git/config files always start with [core]
+scope: source.gitconfig
+
+variables:
+  escape_char: \\"bnt
+  section_name: '[\w.-]+(?=[\s\]])'  # lookahead catches missing space, e.g. color"branch".
+  space_or_eol: (?=(?:\s|$))         # makes values on last line match properly
+  variable_name: '[a-zA-Z][\w-]*'
+  zero_to_255: 25[0-5]|2[0-4][0-9]|1\d\d|[1-9][0-9]|[0-9]
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: section
+
+  boolean:
+    # 0 and 1 are omitted due to mismatches (e.g. "log -1", "HEAD~1"). They will
+    # be matched as unquoted strings instead.
+    - match: \b(?:yes|true|on|off|no|false){{space_or_eol}}
+      scope: constant.language.gitconfig
+
+  color:
+    - include: color-named      # red, blue, green
+    - include: color-attribute  # bold, italic, underline
+    - include: color-numeric    # 0-255
+
+  color-attribute:
+    - match: \b(?:no-?)?(?:ul|strike|reverse|italic|dim|bold|blink){{space_or_eol}}
+      scope: support.constant.color-attribute.gitconfig
+
+  color-named:
+    - match: \b(?:yellow|white|red|normal|magenta|green|cyan|blue|black|auto){{space_or_eol}}
+      scope: support.constant.color.gitconfig
```

Comment:
Why not `constant.language.color.name.gitconfig`? Shold we really use the `support.` here?